.github/workflows/release.yml

```diff
@@ -173,18 +173,19 @@ jobs:
 
   build-linux-amd64-artifacts:
     name: Build linux-amd64 artifacts
-    if: ${{ inputs.build_linux_amd64_artifacts || github.event_name == 'push' || github.event_name == 'schedule' }}
+    if: ${{ inputs.build_linux_amd64_artifacts || github.event_name == 'push' || github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && inputs.publish_github_release && (inputs.build_linux_amd64_artifacts || inputs.build_linux_arm64_artifacts || inputs.build_macos_artifacts || inputs.build_windows_artifacts)) }}
```

Comment:
Do we need to ensure both arm64 and amd64 platforms are selected for Linux platform? We build the multi-platform docker image by default. Missing a platform may cause issues in the release image step.